src/components/file-uploader/_file-uploader.scss

```diff
@@ -152,13 +161,42 @@
     align-items: center;
     justify-content: space-between;
     min-height: rem(30px);
-    background-color: $inverse-01;
+    background-color: $ibm-colors__gray-10;
```

Comment:
@laurenmrice Is there a theme token that should be applied to selected file's background color? Thanks!

Reply:
The background color of the upload should be color token ui-01. There is not a selected state for the uploaded file.

Reply:
@laurenmrice just to clarify, I believe this is not for a selected state. It just refers to the file(s) currently stored in the uploader input

Reply:
Oh okay yeah, all of the files that have been uploaded should all have a background of field-01 (#F3F3F3)

Reply:
@laurenmrice just fixed it, the link should be working again now